pkg/apis/samples/v1alpha1/samplesource_defaults_test.go

```diff
@@ -48,6 +49,7 @@ func TestSampleSourceDefaults(t *testing.T) {
 				Spec: SampleSourceSpec{
 					ServiceAccountName: "default",
 					Interval:           "10s",
+					MessageTemplate:    "Hello, world!",
```

Comment:
Not necessary, but I already have a test to make sure it gets populated so I don't want this test to fail as well for that case - it would just be noisy